vnext/Microsoft.ReactNative.Managed.CodeGen/CodeGenerator.cs

```diff
@@ -18,10 +18,12 @@ namespace Microsoft.ReactNative.Managed.CodeGen
   /// </summary>
   public partial class CodeGenerator
   {
-    private readonly ReactTypes m_reactTypes;
+    public readonly ReactTypes m_reactTypes;
 
     private readonly string m_rootNamespace;
 
+    internal ReactTypes ReactTypes => m_reactTypes;
```

Comment:
>ReactTypes [](start = 24, length = 10)

Why is it needed if we already have a public field?

Reply:
Refactoring snaffu :) Let me fix this.